### PR TITLE
(devx) Auto sort imports

### PR DESCRIPTION
**Pull request by tstirrat**
Uses `module` sorting. there are others, but this closely matches g3 style:

https://github.com/renke/import-sort/tree/master/packages/import-sort-style-module

### CODE REVIEW

File: src/components/Database/DataViewer/InlineQuery.tsx [hunk 1]
```diff
@@ -14,26 +14,27 @@
...
-  QueryParams,
-  DEFAULT_PAGE_SIZE,
-  DEFAULT_QUERY_PARAMS,
-  jsonIshValue,
-} from './common/view_model';
-import { Select } from '@rmwc/select';
-import { TextField } from '@rmwc/textfield';
-import { useState, useEffect, useRef } from 'react';
-import keycode from 'keycode';
+import './InlineQuery.scss';
```
Comment (tjlav5) [inline src/components/Database/DataViewer/InlineQuery.tsx]: DId this get auto-moved to the top?

File: src/components/Database/Database.tsx [hunk 2]
```diff
@@ -14,21 +14,23 @@
  * limitations under the License.
  */
 
-import React, { useState, useEffect, useCallback } from 'react';
-import * as firebase from 'firebase/app';
 import 'firebase/database';
```
Comment (tjlav5) [inline src/components/Database/Database.tsx]: Same question here, is this after sorting?

File: src/components/Database/DataViewer/InlineQuery.tsx (hunk 1 above)
Comment (tstirrat) [reply src/components/Database/DataViewer/InlineQuery.tsx]: yep, side effect imports (no variable assignment) are put at top.

File: src/components/Database/Database.tsx (hunk 2 above)
Comment (tstirrat) [reply src/components/Database/Database.tsx]: Yep, side effects go to the top, these are in the api.ts, so likely don't need to be here anyway

**Pull request merged**